.github/actions/custom-actions/action.yml

```
@@ -45,6 +45,9 @@ runs:
     - name: TeGuio Colombia Staging release custom action
       if: inputs.helpline-name == format('CO_STG')
       uses: ./.github/actions/custom-actions/teguio_staging_custom
+    - name: TeGuio Colombia Production release custom action
+      if: inputs.helpline-name == format('Te Guío CO Production release')
```

```suggestion
      if: inputs.helpline-name == format('CO_PROD')
```

You could also do 

```suggestion
      if: github.workflow == format('Te Guío CO Production release')
```

But the first one is more in line with what we want to do in the future. In fact if you wanted to migrate the other production custom action checks for production helplines over to checking ìnput.helpline-name` that would save somebody a job in the future, but np if you prefer not to

---

@stephenhand  alright! is this the only change needed? I updated the other values per your comment, but I'm unsure if I need to update anything else.  (I don't fully understand these workflows, I don't know for example from where the inputs.helpline-name is getting its value :) )